### PR TITLE
News revamp

### PR DESCRIPTION
**Pull request by macfeaster**
Revamped UI for news. Still needs filtering logic (no itemType filtering/only posts/only events), paging logic (+ UI), and single page logic and view.

### CODE REVIEW

File: src/components/App.css [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
-  padding: 0;
-  font-family: Lato, sans-serif;
+    margin: 0;
+    padding: 0;
+    font-family: Lato, sans-serif;
 }
+
+h3, .cerise #content h3 {
+    border: none;
+}
```
Comment (addem1234) [inline src/components/App.css]: Doesn't follow .editorconfig 😣

File: src/components/Default/index.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const Default = ({ location, lang }) =>
           <div className="row">
             <div className="header-left col-md-2">
               <Link to="/">
-                &laquo;
+                &laquo;&nbsp;
```
Comment (addem1234) [inline src/components/Default/index.js]: Nbsp feels hacky to me, personally I prefer `{' '}`

File: src/components/News/News.module.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    text-align: center;
+}
+
+.gdocs {
+    background: #ae5751;
+}
+
+.fb {
+    background: #43629b;
+}
```
Comment (addem1234) [inline src/components/News/News.module.css]: Editorconfig again 🙄

File: src/components/News/index.js [hunk 4]
```diff
@@ -47,43 +51,146 @@ export const News = ({ location, lang }) =>
...
-                      {{
-                        en: item.titleEnglish,
-                        sv: item.titleSwedish
-                      }}
-                    </Translate>
-                  </h3>
-                  <div>
+          <div className="row">
+            <div className="col-sm-4 col-md-3">
+              <div id="secondary-nav"><h3><a href="/nyheter">Nyheter/Event</a></h3>
```
Comment (addem1234) [inline src/components/News/index.js]: Missing translations here

**Comment by macfeaster** [comment]
Should we implement single item view before merging?

**Comment by addem1234** [comment]
I would say yes. The component should probably be split in a couple of parts as well 😅

**Pull request merged**